File: coding_round2_struct/Trapping_Rain_Water/starter_code/solution.py
import sys

def main():
    data = sys.stdin.read().strip().split()
    if not data:
        return
    n = int(data[0]); h = list(map(int, data[1:1+n]))
    # TODO: compute trapped water and print

if __name__ == "__main__":
    main()

File: coding_round2_struct/Trapping_Rain_Water/starter_code/solution.java
import java.util.*;
public class Main {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        if(!sc.hasNextInt()) return;
        int n = sc.nextInt();
        long[] h = new long[n];
        for(int i=0;i<n;i++) h[i]=sc.nextLong();
        // TODO: compute trapped water and print
    }
}
